s64da_benchmark_toolkit/prepare.py

```diff
@@ -44,24 +49,27 @@ class PrepareBenchmarkFactory:
     CLUSTER_SPEC = {}
 
     def __init__(self, args, benchmark):
-        schemas_dir = os.path.join(s64_benchmark_toolkit_root_dir, benchmark.base_dir, "schemas")
+        schemas_dir = os.path.join(
```

Comment:
Can you please undo the formatting changes? They just make the PR look bigger than it is, and they are probably not consistent with the styling of the rest of the code.
We should probably tackle styling of the code, but it should be done independently.

Reply:
Done. I hoped to do formatting gradually.